src/main/scala/verification/schedulers/DPOR.scala

```diff
@@ -203,8 +203,6 @@ class DPOR extends Scheduler with LazyLogging {
 
           } else {
             val next @ (Unique(MsgEvent(snd, rcv, msg), id), cell, env) = queue.dequeue()
-            logger.trace( Console.GREEN + "Now playing pending: " +
```

Comment:
The changes to RandomScheduler look fine.

On Mon, Mar 9, 2015 at 3:25 PM Colin Scott <notifications@github.com> wrote:

> In src/main/scala/verification/schedulers/DPOR.scala
> <https://github.com/NetSys/sts2-interposition/pull/68#discussion_r26083027>
> :
>
> > @@ -203,8 +203,6 @@ class DPOR extends Scheduler with LazyLogging {
> >
> >            } else {
> >              val next @ (Unique(MsgEvent(snd, rcv, msg), id), cell, env) = queue.dequeue()
> > -            logger.trace( Console.GREEN + "Now playing pending: " +
>
> Whoops! Will put this back in
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NetSys/sts2-interposition/pull/68/files#r26083027>.
>